### PR TITLE
feat: Add new BaseStore logo

### PR DESCRIPTION
**Pull request by israelswf**
## What's the purpose of this pull request?

This PR aims to add the brand new BaseStore logo to the website.

|-|before|after|
|-|-|-|
|Navbar|image|image|
|Footer|image|image|
|Favicon|image|image|

## How does it work?

Just run the preview and see the the logo applied (try to clear the cache if the new logo does not appear) 

## How to test it?

Nothing special to be tested in this PR. When run this locally, remember to run `yarn clean` before `yarn develop`, so any old images and favicon would be properly replaced.

## References

Related [Jira Task](https://vtex-dev.atlassian.net/browse/FSSS-227?atlOrigin=eyJpIjoiYzY1ZGI3Njk4NTA1NDU5OTg1Yjg0MDcyMTk2NTFkMTUiLCJwIjoiaiJ9)

[CSS Tricks: Accessible SVGs (Inline)](https://css-tricks.com/accessible-svgs/#aa-2-inline-svg) (via Heitor ✨ )

[TinyPNG to Compress PNG](https://tinypng.com/)

## Checklist

- [x] CHANGELOG entry added
- [x] Add logo to Navbar
- [x] Add logo to Footer
- [x] Change Favicon

### CODE REVIEW

**Review by danzanzini — APPROVED** [review]

File: src/components/common/Footer/Footer.tsx [hunk 1]
```diff
@@ -89,10 +89,10 @@ function Footer() {
         <UIIcon
           component={
             <img
-              src="/FastStore.png"
-              alt="FastStore icon"
+              src="/BaseStore.png"
+              alt="BaseStore icon"
```
Comment (filipewl) [inline src/components/common/Footer/Footer.tsx:93]:
```suggestion
              alt="BaseStore logo"
```

**Comment by filipewl** [comment]
Maybe this is also a good opportunity to update the copy from FastStore to BaseStore where it makes sense. 

Here are some occurrences where I think it makes sense to update:
CleanShot 2022-04-04 at 10 50 16

What do you think?

**Comment by israelswf** [comment]
> Maybe this is also a good opportunity to update the copy from FastStore to BaseStore where it makes sense.
> 
> Here are some occurrences where I think it makes sense to update: CleanShot 2022-04-04 at 10 50 16
> 
> What do you think?

Thank you, Wands. I was just updating those.

**Review by ArthurTriis1 — APPROVED** [review]

**Review by filipewl — APPROVED** [review]

**Review by eduardoformiga — APPROVED** [review]
LGTM!
I wonder if should we ask for the SVG versions instead of PNG for the sake of decreasing the final bundle.
Some thoughts comparing the `bundle-stats` from base x preview.
1) and 2) the initial JS decrease, which is good for the TBT.
3), 4), 5) we have 1 more asset that increases our final bundle.

I think that now Gatsby is loading the logo in a new request instead of inlining it as base64.


.

**Comment by israelswf** [comment]
> > 
> 
> Lighter in SVG, indeed. Tks, Forms. image

Just documenting here, is looks like we need some further configs to be able to import SVGs. Doing some research about that  with @eduardoformiga .
image

File: src/components/ui/Logo/Logo.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import logoImg from './logo.png'
 
 function Logo() {
-  return <img src={logoImg} alt="Faststore logo" />
+  return <img src={logoImg} alt="BaseStore logo" />
```
Comment (heitorado) [inline src/components/ui/Logo/Logo.tsx:6]: Not sure if we are avoiding to do this, but maybe this does the trick?

I've already made the SVG logo accessible. This is what I used for reference: [CSS Tricks: Accessible SVGs (Inline)](https://css-tricks.com/accessible-svgs/#aa-2-inline-svg)

```suggestion
  return (
    <svg
      width="176"
      height="38"
      viewBox="0 0 176 38"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      aria-labelledby="logoTitle logoDesc"
      role="img"
    >
      <title id="logoTitle">BaseStore logo</title>
      <desc id="logoDesc">
        The store logo with with a stylized letter B in a dark blue gradient
        background
      </desc>
      <circle cx="19" cy="19" r="19" fill="url(#paint0_linear_2299_155683)" />
      <mask
        id="mask0_2299_155683"
        style={{ maskType: 'alpha' }}
        maskUnits="userSpaceOnUse"
        x="0"
        y="0"
        width="38"
        height="38"
      >
        <circle cx="19" cy="19" r="19" fill="url(#paint1_linear_2299_155683)" />
      </mask>
      <g mask="url(#mask0_2299_155683)">
        <path
          d="M14.9639 14.9555L26.1062 12.4082L6.22236 39.3474L-4 39.3475L14.9639 14.9555Z"
          fill="url(#paint2_linear_2299_155683)"
        />
        <path
          d="M-3.89694 27.392L14.9639 3V14.9555L-3.89694 39.3475V27.392Z"
          fill="#B7C8F2"
        />
        <path
          d="M14.9639 3H20.5647C24.0743 3 26.9194 5.8451 26.9194 9.35471C26.9194 12.4479 24.4119 14.9555 21.3186 14.9555H14.9639V3Z"
          fill="#E7EEFF"
        />
        <path
          d="M15.1961 31.8844L30 28.5L3.58174 64.2924L-10 64.2924L15.1961 31.8844Z"
          fill="url(#paint3_linear_2299_155683)"
        />
        <path
          d="M-9.86307 48.408L15.1961 16V31.8845L-9.86307 64.2924V48.408Z"
          fill="#B7C8F2"
        />
        <path
          d="M15.1961 16H23.1383C27.5247 16 31.0805 19.5559 31.0805 23.9422V24.8845C31.0805 28.7505 27.9465 31.8845 24.0805 31.8845H15.1961V16Z"
          fill="#E7EEFF"
        />
      </g>
      <path
        d="M47 23.348H54.952C59.74 23.348 62.148 21.276 62.148 17.832C62.148 15.172 60.468 13.436 57.584 13.128V13.1C60.104 12.512 61.504 10.86 61.504 8.9C61.504 6.016 59.572 4.308 55.176 4.308H47V23.348ZM50.528 20.548V14.864H54.784C57.612 14.864 58.564 15.928 58.564 17.776C58.564 19.708 57.304 20.548 54.504 20.548H50.528ZM50.528 12.232V7.08H54.672C56.884 7.08 57.92 8.004 57.92 9.628C57.92 11.28 56.744 12.232 54.532 12.232H50.528Z"
        fill="#1A3470"
      />
      <path
        d="M68.0109 23.628C70.3909 23.628 72.5189 22.284 73.2189 19.204H73.2749V23.348H76.4389V15.228C76.4389 11.196 74.0589 9.18 70.2789 9.18C66.7509 9.18 64.5109 10.804 64.0349 13.38L67.0309 14.36C67.2829 12.652 68.5429 11.812 70.1109 11.812C71.9309 11.812 73.0789 12.736 73.1069 14.808L69.6349 15.256C66.3589 15.648 63.5589 16.6 63.5589 19.68C63.5589 22.34 65.4629 23.628 68.0109 23.628ZM66.9749 19.344C66.9749 17.888 68.3189 17.524 70.0829 17.3L73.1629 16.88C73.0789 18.868 71.5109 21.052 69.0469 21.052C67.8429 21.052 66.9749 20.52 66.9749 19.344Z"
        fill="#1A3470"
      />
      <path
        d="M84.9831 23.628C88.8471 23.628 91.2271 21.948 91.2271 18.952C91.2271 16.012 88.6231 15.452 86.3271 15.06L84.4511 14.78C83.1351 14.556 82.2951 14.248 82.2951 13.212C82.2951 12.092 83.2471 11.532 84.6191 11.532C86.2431 11.532 87.3071 12.316 87.4751 13.94L90.7231 13.436C90.4151 10.496 87.7831 9.18 84.7311 9.18C81.4831 9.18 78.9351 10.748 78.9351 13.464C78.9351 15.984 80.9791 16.824 82.9111 17.132L85.0671 17.468C86.8031 17.748 87.7551 18.084 87.7551 19.26C87.7551 20.436 86.8031 21.164 84.9551 21.164C83.2471 21.164 82.0711 20.352 81.8751 18.504L78.5151 19.064C78.7111 22.088 81.2311 23.628 84.9831 23.628Z"
        fill="#1A3470"
      />
      <path
        d="M99.4466 23.628C102.975 23.628 105.187 21.948 105.803 19.232L102.667 18.56C102.331 20.268 101.183 21.052 99.5586 21.052C97.2906 21.052 96.1426 19.568 96.0586 16.712H105.859C105.859 16.516 105.887 16.376 105.887 15.9C105.887 11.532 103.423 9.18 99.5306 9.18C95.1906 9.18 92.6146 12.204 92.6146 16.6C92.6146 20.912 95.1066 23.628 99.4466 23.628ZM96.1986 14.724C96.5346 12.624 97.6546 11.532 99.4466 11.532C101.267 11.532 102.303 12.68 102.415 14.724H96.1986Z"
        fill="#1A3470"
      />
      <path
        d="M115.33 23.656C119.698 23.656 122.806 21.472 122.806 17.608C122.806 13.912 120.286 12.708 115.582 12.008C112.782 11.56 111.802 10.972 111.802 9.292C111.802 7.78 113.09 6.94 115.05 6.94C117.234 6.94 118.69 7.724 119.222 10.328L122.19 9.18C121.574 5.596 118.942 4 115.134 4C110.962 4 108.274 6.212 108.274 9.572C108.274 12.96 110.542 14.248 115.106 14.948C117.766 15.368 119.166 15.9 119.166 17.804C119.166 19.568 117.71 20.716 115.498 20.716C113.034 20.716 111.298 19.848 110.85 16.824L107.462 17.72C107.826 21.78 110.682 23.656 115.33 23.656Z"
        fill="#1A3470"
      />
      <path
        d="M129.923 23.628C130.931 23.628 131.995 23.488 132.751 23.32L133.115 20.576C132.583 20.716 131.799 20.884 130.903 20.884C129.783 20.884 129.167 20.52 129.167 19.148V12.092H132.975V9.46H129.167V5.652H126.787C126.591 8.424 125.723 9.544 123.455 10.02V12.092H125.807V19.904C125.807 22.676 127.347 23.628 129.923 23.628Z"
        fill="#1A3470"
      />
      <path
        d="M141.421 23.628C145.369 23.628 148.533 21.136 148.533 16.404C148.533 11.672 145.369 9.18 141.421 9.18C137.501 9.18 134.309 11.672 134.309 16.404C134.309 21.136 137.501 23.628 141.421 23.628ZM137.781 16.404C137.781 13.52 139.237 11.924 141.421 11.924C143.633 11.924 145.061 13.52 145.061 16.404C145.061 19.26 143.633 20.912 141.421 20.912C139.237 20.912 137.781 19.26 137.781 16.404Z"
        fill="#1A3470"
      />
      <path
        d="M151.034 23.348H154.394V16.572C154.394 13.828 155.682 12.456 158.09 12.456C158.678 12.456 159.35 12.512 159.742 12.624L159.714 9.404C159.378 9.292 158.762 9.18 158.258 9.18C156.242 9.18 154.954 10.104 154.226 13.912H154.142V9.46H151.034V23.348Z"
        fill="#1A3470"
      />
      <path
        d="M167.024 23.628C170.552 23.628 172.764 21.948 173.38 19.232L170.244 18.56C169.908 20.268 168.76 21.052 167.136 21.052C164.868 21.052 163.72 19.568 163.636 16.712H173.436C173.436 16.516 173.464 16.376 173.464 15.9C173.464 11.532 171 9.18 167.108 9.18C162.768 9.18 160.192 12.204 160.192 16.6C160.192 20.912 162.684 23.628 167.024 23.628ZM163.776 14.724C164.112 12.624 165.232 11.532 167.024 11.532C168.844 11.532 169.88 12.68 169.992 14.724H163.776Z"
        fill="#1A3470"
      />
      <path
        d="M49.485 35.077C50.577 35.077 51.431 34.426 51.578 33.229L50.878 33.068C50.78 34.006 50.241 34.454 49.499 34.454C48.617 34.454 48.029 33.768 48.029 32.627C48.029 31.472 48.603 30.8 49.499 30.8C50.241 30.8 50.71 31.206 50.864 32.06L51.557 31.92C51.375 30.814 50.584 30.177 49.541 30.177C48.232 30.177 47.322 31.052 47.322 32.627C47.322 34.195 48.218 35.077 49.485 35.077ZM53.459 35H56.399V34.384H54.152V30.24H53.459V35ZM60.011 35.07C61.264 35.07 62.209 34.195 62.209 32.62C62.209 31.045 61.264 30.17 60.011 30.17C58.765 30.17 57.813 31.045 57.813 32.62C57.813 34.195 58.765 35.07 60.011 35.07ZM58.541 32.62C58.541 31.472 59.129 30.793 60.011 30.793C60.9 30.793 61.488 31.472 61.488 32.62C61.488 33.768 60.9 34.447 60.011 34.447C59.129 34.447 58.541 33.768 58.541 32.62ZM65.8062 35.077C66.8492 35.077 67.5982 34.545 67.5982 33.607C67.5982 32.676 66.9332 32.389 65.8762 32.214C65.0992 32.088 64.8052 31.92 64.8052 31.437C64.8052 31.003 65.1692 30.772 65.7152 30.772C66.3172 30.772 66.6952 30.989 66.8072 31.703L67.4302 31.521C67.3182 30.576 66.6672 30.17 65.7362 30.17C64.7492 30.17 64.1122 30.695 64.1122 31.493C64.1122 32.333 64.6652 32.641 65.6802 32.802C66.4502 32.935 66.8772 33.068 66.8772 33.642C66.8772 34.132 66.4642 34.468 65.8272 34.468C65.1762 34.468 64.6862 34.258 64.5672 33.39L63.8952 33.558C63.9932 34.622 64.7142 35.077 65.8062 35.077ZM69.6022 35H72.7382V34.391H70.2952V32.823H72.3812V32.221H70.2952V30.849H72.7312V30.24H69.6022V35ZM74.7349 35H76.2189C77.9129 35 78.7949 34.16 78.7949 32.613C78.7949 31.073 77.9129 30.24 76.2189 30.24H74.7349V35ZM75.4279 34.412V30.828H76.1839C77.5209 30.828 78.0669 31.486 78.0669 32.613C78.0669 33.747 77.5209 34.412 76.1839 34.412H75.4279ZM83.6741 35H85.5221C86.7401 35 87.3281 34.475 87.3281 33.628C87.3281 32.956 86.9221 32.536 86.2081 32.452V32.445C86.7961 32.291 87.1671 31.899 87.1671 31.395C87.1671 30.66 86.6911 30.24 85.6201 30.24H83.6741V35ZM84.3671 34.419V32.809H85.5431C86.3551 32.809 86.6141 33.117 86.6141 33.635C86.6141 34.188 86.2501 34.419 85.4451 34.419H84.3671ZM84.3671 32.242V30.814H85.5291C86.1661 30.814 86.4531 31.073 86.4531 31.521C86.4531 31.983 86.1101 32.242 85.4801 32.242H84.3671ZM89.3605 35H92.4965V34.391H90.0535V32.823H92.1395V32.221H90.0535V30.849H92.4895V30.24H89.3605V35ZM95.6334 35H96.3264V30.821H97.8174V30.24H94.1424V30.821H95.6334V35ZM101.343 30.24H100.405L98.7952 35H99.5512L99.9012 33.873H101.826L102.176 35H102.946L101.343 30.24ZM100.09 33.278L100.86 30.793L101.637 33.278H100.09Z"
        fill="#1A3470"
      />
      <defs>
        <linearGradient
          id="paint0_linear_2299_155683"
          x1="12"
          y1="20.5"
          x2="40.5"
          y2="4"
          gradientUnits="userSpaceOnUse"
        >
          <stop offset="0.0677083" stopColor="#1C3670" />
          <stop offset="1" stopColor="#B6C6ED" />
        </linearGradient>
        <linearGradient
          id="paint1_linear_2299_155683"
          x1="19"
          y1="-4.02331e-07"
          x2="32.5"
          y2="-9.5"
          gradientUnits="userSpaceOnUse"
        >
          <stop stopColor="#001947" />
          <stop offset="1" stopColor="#001947" stopOpacity="0" />
        </linearGradient>
        <linearGradient
          id="paint2_linear_2299_155683"
          x1="15.569"
          y1="13.9135"
          x2="20.361"
          y2="19.778"
          gradientUnits="userSpaceOnUse"
        >
          <stop stopColor="#9FB8EF" />
          <stop offset="0.934068" stopColor="#0A1C46" />
        </linearGradient>
        <linearGradient
          id="paint3_linear_2299_155683"
          x1="16"
          y1="30.5"
          x2="22.3667"
          y2="38.2918"
          gradientUnits="userSpaceOnUse"
        >
          <stop stopColor="#9FB8EF" />
          <stop offset="0.934068" stopColor="#0A1C46" />
        </linearGradient>
      </defs>
    </svg>
  )
```

Comment (israelswf) [reply src/components/ui/Logo/Logo.tsx:6]: Brilliant, Heitor ✨  Thank you!

**Pull request merged**